src/NetAppFiles/NetAppFiles/ChangeLog.md

```diff
@@ -18,6 +18,15 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added SecurityOperators to ActiveDirectory
+* Volume list now gets all volumes instead of first over 100 volumes
+* Added SnapshotPolicyId to NewAzNetAppFilesVolume to set snapshot policy to volume
+
+
+## Version 0.4.0
```

Comment:
Why you have 0.4.0 here?
https://www.powershellgallery.com/packages/Az.NetAppFiles/0.3.0
https://github.com/Azure/azure-powershell/blob/master/src/NetAppFiles/NetAppFiles/ChangeLog.md

Reply:
This bumps the api-version used for the REST API, of and would mean bump in version number does it not?  What should it be?

Reply:
The versions here represents for the previous released versions of NetAppFiles. All the changes after the latest previous release should be put under upcoming release section and will be composed into next release note.

Please just remove 0.4.0 here.